test/create-new-user.test.js

```diff
@@ -0,0 +1,43 @@
+const Lab = require('lab')  
+const Code = require('code')  
+const Hapi = require('hapi')
+
+const lab = (exports.lab = Lab.script())
+const { describe, it } = lab
+const expect = Code.expect
+
+const createNewUserHandler = require('../src/route-handlers/create-new-user')
+const server = new Hapi.Server()
+
+lab.experiment('Create new user', () => {
+
+  lab.beforeEach(() => {
+    
+    server.route({
+      method: 'POST',
+      path: '/',
+      handler: createNewUserHandler
+    })
+  })
+
+  lab.test('should generate a system guid', async () => {
+
+    // these must match the route you want to test
+    const injectOptions = {
+      method: 'POST',
+      url: '/',
+      payload: {
+        name: "Beverly Benedict",
```

Comment:
Yes. All of my current work projects install an internal package that contains our base linting rules, code formatting rules, and helper webpack/rollup configs. They run on local build, on git hook, and cause remote builds to fail if linting rules don't pass. 

The convenience of them being part of our base repo intialization template has probably spoiled me to the point that I didn't even think about installing them here. Clearly, that should have happened. Its very helpful to have standard code formatting, and code linting.

Reply:
Yes, it's very helpful; so much so, I can see how it could be forgotten.  Honestly, we should be to a point where we dont need to think about formatting on input, as it should be authoritatively stored formatted as such.  Go started this trend with gmfmt and Im happy to see other following suit!